index.js

```diff
@@ -76,12 +81,18 @@ finalScore(inning, 9) might return:
 
 */ 
 
-function finalScore(/*code Here*/){
-
-  /*Code Here*/
+function finalScore(inning,inningNum){
+ let homeScr = 0;
+ let awayScr = 0;
+ for(let i=0; i<inningNum -1; ++i) {
+   homeScr += inning();
+   awayScr += inning();
+ }
+ return `Home: ${homeScr}, Away: ${awayScr}`;
 
-}
+};
 
+finalScore(inning,9);
 /* Task 4: 
 
 Create a function called `scoreboard` that accepts the following parameters: 
```

Comment:
It doesn't seem like you've met MVP by completing all of the tasks in index.js :(